101-bundle-json.md

```diff
@@ -764,6 +767,7 @@ The above declares three actions: `io.cnab.status`, `io.cnab.migrate` and `io.cn
 
 Each action is accompanied by a description, which contains the following fields:
 
+- `title`: The human-readable name of the operation. If omitted, tools MAY use the value of the `actions` key (e.g. `io.cnab.dry-run`). User agents that display the title SHOULD disambiguate if two or more actions have the same `title` value.
```

Comment:
Should we add `(OPTIONAL)` at the end of this entry, as is done elsewhere?  (Technically it looks like strict adherence isn't already in place as `modifies` below also doesn't have this...)

Reply:
Yes. Good catch.